config/foreman.hiera/family/RedHat-8.yaml

```diff
@@ -1,4 +1,8 @@
 ---
+# EL8 only due to issues with older Apache event mpm on EL7. When EL7
+# support is dropped, this should move to tuning/common.yaml
+apache::mpm_module: 'event'
```

Comment:
Would setting `apache::mpm_module: 'event'` in `config/foreman.hiera/common.yaml` (or `tuning/common.yaml`, idk what's better) and then `apache::mpm_module: 'prefork'` in `config/foreman.hiera/family/RedHat-7.yaml` get us the same result?

The benefit would be that this then also would apply to Debian/Ubuntu installations.